addons/website_profile/views/website_profile.xml

```diff
@@ -283,7 +283,7 @@
                             <div class="o_wprofile_email_validation_container mb16 mt16">
                                 <t t-call="website_profile.email_validation_banner">
                                     <t t-set="redirect_url" t-value="'/profile/user/%s' % user.id"/>
-                                    <t t-set="send_validation_email_message">Click here to send a verification email.</t>
+                                    <t t-set="send_validation_email_message">.</t>
```

Comment:
Strange to see a message with just a `.` here, can you just give false here and add the dot (or `!`) in the parent template ?